Assets/MixedRealityToolkit.Services/InputSystem/MixedRealityInputSystem.cs

```diff
@@ -1056,19 +959,12 @@ private void HandleClick()
 
         #region Pointer Up
 
-        private static readonly ExecuteEvents.EventFunction<IMixedRealityPointerHandler> OnPointerUpEventHandler =
-            delegate (IMixedRealityPointerHandler handler, BaseEventData eventData)
-            {
-                var casted = ExecuteEvents.ValidateEventData<MixedRealityPointerEventData>(eventData);
-                handler.OnPointerUp(casted);
-            };
-
         /// <inheritdoc />
         public void RaisePointerUp(IMixedRealityPointer pointer, MixedRealityInputAction inputAction, Handedness handedness = Handedness.None, IMixedRealityInputSource inputSource = null)
         {
             pointerEventData.Initialize(pointer, inputAction, handedness, inputSource);
 
-            HandlePointerEvent(pointerEventData, OnPointerUpEventHandler);
+            HandlePointerEvent(pointerEventData, MixedRealityEventHandlers.OnPointerUpEventHandler);
```

Comment:
Oh, I see, it looks like you moved all of these into their own file. Any reason for this? Just to make this file shorter? I'm a bit wary to see a big change like this in an experimental feature, but if it's just moving code around I suppose it's okay.